### PR TITLE
Added first test that relies on creating a new world class that is empty

### PR DESCRIPTION
**Pull request by sethpthomas91**

### CODE REVIEW

**Review by scottyplunkett — CHANGES_REQUESTED** [review]
Nice work getting your first PR in! Please understand feedback in a PR can often be terse, brevity is not meant to offend.   

I noticed you set this project up as a Gem, I probably threw you off with my Gem description on slack. I should have mentioned; **all Gems are ruby projects**, but **not all ruby projects need to be or use gems**. A ruby project could potentially be as simple as a single ruby file (though you _should_ always have a README as well).

Concretely:
1. Adjust the project so that it is **NOT A GEM**
2. For testing, let's use `rspec`.... It's typically the testing framework we use for testing ruby instead of minitest.... [rspec](https://rspec.info/) is a Gem, so your `Gemfile` should look something like the below:
```
source "https://rubygems.org"

gem 'rspec'
```
Don't hesitate to reach out with questions!

**Comment by sethpthomas91** [comment]
Refactored the project to not be a gem. Added two classes, Cell and World. Wrote one test for cell.

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```
Comment (scottyplunkett) [inline Gemfile]: I don't think you need this line unless you have Gem's coming from github sources.

```suggestion
```

File: .rspec [hunk 2]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (scottyplunkett) [inline .rspec:1]: You can delete this file, it's not necessary.

File: lib/cell.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    end
+
+    def make_alive
+        @is_alive = true
+    end
+
+    def kill
+        @is_alive = false
+    end
+
```
Comment (scottyplunkett) [inline lib/cell.rb]:
```suggestion
```

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (scottyplunkett) [inline spec/spec_helper.rb:98]: You can delete this file, it's not necessary.

File: Gemfile [hunk 5]
```diff
@@ -1,6 +1,7 @@
+# frozen_string_literal: true
+
```
Comment (scottyplunkett) [inline Gemfile]: If not necessary, (here it is not, I think) I would not use the `# frozen_string_literal` directive.
```suggestion
```

**Review by scottyplunkett — CHANGES_REQUESTED** [review]
In addition to the comments. I would add the `README` back to the repo with basic setup, dependencies/requirements (i.e. ruby version) & test running instructions. I would also delete the `runner.rb` until you're using it. 

Also use 2 spaces instead of 4 for indentation, thanks!

**Pull request merged**